### ISSUE TITLE
Definition for rule 'react/no-unstable-nested-components' was not found 

### ISSUE THREAD

**Issue by karlhorky** [upleveled/eslint-config-upleveled#49]
Currently, there is an error with the ESLint config, where it is complaining that a certain ESLint rule was not found:

```
Definition for rule 'react/no-unstable-nested-components' was not found
```

The solution is to add a `resolutions` field (for [Yarn Resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/)) to the `package.json`:

```
  "resolutions": {
    "eslint-plugin-react": "7.23.2"
  }
```

**Comment by karlhorky** [comment]
@Josehower can you add a new line to the setup commands for this?

**Comment by karlhorky** [comment]
Fixed in https://github.com/upleveled/eslint-config-upleveled/pull/50
